pkgs/by-name/op/openfga-cli/package.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+}:
+
+buildGoModule rec {
+  pname = "openfga-cli";
+  version = "0.4.0";
```

```suggestion
let
  pname = "openfga-cli";
  version = "0.4.0";
in
buildGoModule {
```

Using `rec` is discouraged, you can learn more about it here: https://nix.dev/guides/best-practices#recursive-attribute-set-rec